daily-planner/lib/summaryService.ts

```
@@ -0,0 +1,288 @@
+    }
+  }
+
+  async forceSummaryGeneration(type: 'weekly' | 'monthly' | 'yearly', date: string): Promise<Summary | null> {
+    try {
+      switch (type) {
+        case 'weekly':
+          return await this.generateWeeklySummary(date);
+        case 'monthly':
+          return await this.generateMonthlySummary(date);
+        case 'yearly':
+          const year = parseInt(date.split('-')[0]);
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)